Data-Structures/src/main/java/datastructures/tree/BinaryTree.java

```diff
@@ -0,0 +1,53 @@
+package datastructures.tree;
+
+import java.util.ArrayList;
+
+public class BinaryTree {
+        ArrayList<Integer> preOrder = new ArrayList<>();
+//        int [] preOrderArray;
+        ArrayList<Integer> inOrder = new ArrayList<>();
+        ArrayList<Integer> postOrder = new ArrayList<>();
```

Comment:
The BinaryTree class should not have these ArrayLists as member variables. All your BinaryTree needs is a `Node root`